### ISSUE TITLE
Remove Vertical extents

### ISSUE THREAD

**Issue by mguarinello** [northwest-knowledge-network/mdedit#45]
Need to figure out how to let users indicate the units for vertical extent. ISO seems to have "verticalCRS" which is not quite the same thing.
I will hunt up what option might work and then let @mtpain know what field to add and switch the assignment at that time.

**Comment by mguarinello** [comment]
Matt will add a field. Marisa still needs to figure out where it will go in the XML.

**Comment by flathers** [comment]
This used to be easy.  In ISO 19115:2003(E), the EX_VerticalExtent element contained an element called unitOfMeasure that was a simple text field that we could put 'meters' or 'feet' or whatever into.  At some point, that mechanism was replaced by a required complex element called verticalCRS that requires a lot more effort to fill out.  In the Challis LiDAR metadata, I just used '<gmd:verticalCRS gco:nilReason="unknown"/>' as a stub for it.

So there's no simple way to express a unit of measurement for the vertical range.  Also, the vertical extent is of limited use in searching and probably doesn't add very much to a person's understanding of the data when they're viewing the summary.  I suggest removing the vertical extent fields entirely, and if someone really wants them, we can help them to get that info into their metadata manually.
